.env.prod.example

```
@@ -0,0 +1,37 @@
+# App
+ENVIRONMENT=prod
+APP_NAME=tam-api
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_NAME key should go before the ENVIRONMENT key

(UnorderedKey)